akka-http-cors/src/test/scala/ch/megard/akka/http/cors/scaladsl/settings/CorsSettingsSpec.scala

```diff
@@ -0,0 +1,77 @@
+package ch.megard.akka.http.cors.scaladsl.settings
+
+import akka.http.scaladsl.model.HttpMethods
+import akka.http.scaladsl.model.headers.{HttpOrigin, HttpOriginRange}
+import ch.megard.akka.http.cors.scaladsl.model.HttpHeaderRange
+import com.typesafe.config.{ConfigException, ConfigFactory, ConfigValueFactory}
+import org.scalatest.{Matchers, WordSpec}
+
+class CorsSettingsSpec extends WordSpec with Matchers {
+
+  import HttpMethods._
+
+  private val validConfigStr =
+    """
+      |akka.http.cors {
+      |  allow-generic-http-requests = true
+      |  allow-credentials = true
+      |  allowed-origins = "*"
+      |  allowed-headers = "*"
+      |  allowed-methods = ["GET", "POST", "HEAD", "OPTIONS"]
+      |  exposed-headers = []
+      |  max-age = 30 minutes
+      |}
+    """.stripMargin
+
+  private val validConfig = ConfigFactory.load(validConfigStr)
+
+  "The CorsSettings object" should {
```

Comment:
Missing the test about reading the values using the list `[...]` syntax.

Reply:
doesn't the initial test covers this syntax on `allowed-methods`?